canonical/ubuntu.com
# Redeem Activation Keys (copy over from abhigyanghosh30:redeem_beta_invites)

## Done

- Beta participants can redeem their invitation codes

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/
    - Be sure to test on mobile, tablet and desktop screen sizes
- Generate activation key from the back office tool
    - Go to /credentials/redeem/{activationKey}
    - It should attach an exam to the user
- Also visit /credentials/shop/manage to ensure that the purchased keys show up in the table
    - Make sure key rotation works
    - Make sure copying key to clipboard works

## Issue / Card

Fixes [https://warthogs.atlassian.net/browse/CRED-230](https://warthogs.atlassian.net/browse/CRED-230)

## Screenshots

[If relevant, please include a screenshot.]


## Help

[QA steps](https://discourse.canonical.com/t/qa-steps/152) - [Commit guidelines](https://discourse.canonical.com/t/commit-guidelines/148)
